sql/core/src/main/scala/org/apache/spark/sql/execution/exchange/ValidateRequirements.scala

```diff
@@ -30,6 +30,14 @@ import org.apache.spark.sql.execution._
  */
 object ValidateRequirements extends Logging {
 
+  def validate(
+      plan: SparkPlan,
+      requiredDistribution: Distribution,
+      requiredOrdering: Seq[SortOrder]): Boolean = {
+    validate(plan, requiredDistribution) &&
+      SortOrder.orderingSatisfies(plan.outputOrdering, requiredOrdering)
+  }
+
   def validate(plan: SparkPlan, requiredDistribution: Distribution): Boolean = {
```

Comment:
Seems we can remove this now?

Reply:
`OptimizeSkewedJoin` still use this. I considered unify them, but seems `OptimizeSkewedJoin` does not affect the required output ordering.